keps/0001a-meta-kep-implementation.md

```diff
@@ -0,0 +1,217 @@
+|   |   |   |   ├── guide-for-operators.md (required)
+|   |   |   |   └── guide-for-project-maintainers.md
+|   |   |   ├── index.json (required) # used for site generation e.g., Hugo
+|   |   |   ├── metadata.yaml (required) # used for automation / project tracking
+|   |   └── └── OWNERS
+│   ├── sig-foo
+|   ├── ...
+|   └── sig-n
+```
+
+metadata.yaml would contain information that was previously in a KEP's YAML front-matter:
+
+```
+---
+authors: # required
+  - "calebamiles" # just a GitHub handle for now
+  - "jbeda"
+title: "Kubernetes Enhancement Proposal process"
+number: 42 # required
+owning-sig: "sig-pm" # required
+participating-sigs:
+  - "sig-architecture"
+  - "sig-contributor-experience"
+approvers: # required
+  - "bgrant0607" # just a GitHub handle for now
+reviewers:
+  - "justaugustus"  # just a GitHub handle for now
+  - "jdumars"
+editors:
+  - null # generally omit empty/null fields
+status: "active" # required
+github:
+  issues:
+    - null # GitHub url
+  pull_requests:
+    - null # GitHub url
+  projects:
+    - project_id: null
+      card_id: null
+releases: # required
+  - k8s_version: v1.9
+    kep_status: "active"
+    k8s_status: "alpha" # one of alpha|beta|GA
+  - k8s_version: v1.10
+    kep_status: "active"
+    k8s_status: "alpha"
+replaces:
+  - kep_location: null
+superseded-by:
+  - kep_location: null
+created: 2018-01-22 # in YYYY-MM-DD
+updated: 2018-09-04
+```
+
+- Move existing KEPs into [k/features]
+- Create a `kind/kep` label for [k/community] and [k/features]
+  - For `k/community`:
+    - Label incoming KEPs as `kind/kep`
+    - Enable searches of `org:kubernetes label:kind/kep`, so we can identify active PRs to `k/community` and reroute the PR authors to `k/enhancements` (depending on the state)
+  - For `k/enhancements` (fka `k/features`):
+    - Label incoming KEPs as `kind/kep`
+    - Classify KEP submissions / tracking issues as `kind/kep`, differentiating them from `kind/feature`
+- Move existing design proposals into [k/features]
+- Move existing architectural documents into [k/features] (process TBD)
+- Deprecate design proposals
+- Rename [k/features] to [k/enhancements]
+- Create tombstones / redirects to [k/enhancements]
+- Prevent new KEPs and design proposals from landing in [k/community]
+- Remove `kind/kep` from [k/community] once KEP migration is complete
+- Correlate existing Feature tracking issues with links to KEPs
+- Fix [KEP numbering races] by using the GitHub issue number of the KEP tracking issue
+- Coordination of existing KEPs to use new directory structure (with SIG PM guidance per SIG)
+
+#### Visibility and Automation
+
+- Create tooling to:
+  - Generate KEP directories and associated metadata
+  - Present KEPs, through some easy to use mechanism e.g., https://enhancements.k8s.io. This would be a redesigned version of https://contributor.kubernetes.io/keps/. We envision this site / repo having at least three directories:
```

Comment:
We are already building out contributors.k8s.io and we should put the keps there.  That was one of the main reasons for that site.  /cc @castrojo

Reply:
We can totally do this, we just need the repo URL when it's created and we can implement it. Doing it with three sections wouldn't be a problem. Hugo can be sensitive to the metadata though, it'd be great if the tooling could lint it before hugo consumes it.